### ISSUE TITLE
loss 为0 

### ISSUE THREAD

**Issue by wwangxinhao** [OFA-Sys/Chinese-CLIP#236]
您好，我训练自己的数据，loss 为 0 可能是什么原因，日志如下：
2023-12-14,08:40:01 | INFO | Rank 0 | Global Steps: 240/270 | Train Epoch: 3 [60/90 (67%)] | Loss: 0.000000 | Image2Text Acc: 100.00 | Text2Image Acc: 100.00 | Data Time: 0.042s | Batch Time: 0.170s | LR: 0.000004 | logit_scale: 2.659 | Global Batch Size: 1

**Comment by wwangxinhao** [comment]
请教一下知道如何解决这个问题吗，我训练玩的模型分类都是错误的，我是按照文档组织的数据

**Comment by Yang-Arky** [comment]
我也遇到了，请问解决了吗
